Journaley/Forms/MainForm.cs

```diff
@@ -638,53 +639,18 @@ private void OnGetMinMaxInfo(ref Message m)
         /// </summary>
         private void UpdateWebBrowser()
         {
+            string initialString = this.SelectedEntry.EntryText;
+            string docType = "<!DOCTYPE HTML PUBLIC \"-//W3C//DTD HTML 4.01//EN\" \"http://www.w3.org/TR/html4/strict.dtd\">";

```

Comment:
I'm fine with adding the doctype declaration, but why not HTML5? Let's just go with:
```csharp
string docType = "<!DOCTYPE html>";
```